static/app/components/modals/savedSearchModal/createSavedSearchModal.tsx

```diff
@@ -26,7 +26,14 @@ const DEFAULT_SORT_OPTIONS = [
   IssueSortOptions.USER,
 ];
 
-function validateSortOption({sort}: {sort?: string}) {
+function validateSortOption({sort, organization}: {sort?: string}) {
+  const hasBetterPrioritySort = organization.features.includes(
+    'issue-list-better-priority-sort'
+  );
+  const sortOptions = [...DEFAULT_SORT_OPTIONS];
+  if (hasBetterPrioritySort) {
+    sortOptions.push(IssueSortOptions.BETTER_PRIORITY);
+  }
   if (DEFAULT_SORT_OPTIONS.find(option => option === sort)) {
```

Comment:
Shouldn't this be `sortOptions.find(...)`?

Reply:
oops yes!